mmv1/third_party/terraform/tests/data_source_google_cloudfunctions2_function_test.go

```diff
@@ -0,0 +1,75 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceGoogleCloudFunctions2Function_basic(t *testing.T) {
+	t.Parallel()
+
+	funcDataNameHttp := "data.google_cloudfunctions2_function.function_http_v2"
+	functionName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+	bucketName := fmt.Sprintf("tf-test-bucket-%d", randInt(t))
+	zipFilePath := "./test-fixtures/cloudfunctions2/function-source.zip"
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckCloudfunctions2functionDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceGoogleCloudFunctions2FunctionConfig(functionName,
+					bucketName, zipFilePath),
+				Check: resource.ComposeTestCheckFunc(
+					checkDataSourceStateMatchesResourceStateWithIgnores(funcDataNameHttp,
+						"google_cloudfunctions2_function.function_http_v2", map[string]struct{}{"build_config.0.source.0.storage_source.0.bucket": {}, "build_config.0.source.0.storage_source.0.object": {}}),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceGoogleCloudFunctions2FunctionConfig(functionName, bucketName, zipFilePath string) string {
+	return fmt.Sprintf(`
```

Comment:
nit: test configs are generally double-spaced rather than tabs (although the Go source in the rest of the file is done w/ tabs)

Reply:
Sure, I will make these changes.